jenkins/jenkins_home/users/github67746756_4203538913420999439/config.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version='1.1' encoding='UTF-8'?>
```

Comment:
This file should also probably be removed.

Reply:
yes